waveform_editor/gui/io/manager.py

```diff
@@ -55,27 +58,60 @@ def create_new_file(self):
         self.main_gui.load_yaml(yaml_content)
         self.open_file = None
 
+    def _confirm_and_execute(self, action, message):
```

Comment:
Let's rename to `_confirm_unsaved_changes`, because it doesn't ask the user when there are no unsaved changes?